### ISSUE TITLE
Generated HTML docs has no API list

### ISSUE THREAD

**Issue by loretoparisi** [aubm/postmanerator#27]
I'm exporting from Postman both v1 or v2 of Collection json.
Then I run the command with default theme
```
./postmanerator -theme="default" -collection=./api.json -output=./docs/api.html
```

When I open the `api.html` file, I do not any api api reference in. just the basic structure like this
<img width="1077" alt="schermata 2017-05-12 alle 14 39 58" src="https://cloud.githubusercontent.com/assets/163333/25998364/e779928a-3720-11e7-98de-31c63b8f39e7.png">

**Comment by ptramonti** [comment]
I have had the same problems... you have to:
1) Use Postman v1 Collection json
2) Insert all webapi into one (or more) folder inside Postman

Regards,
Paolo

**Comment by loretoparisi** [comment]
@ptramonti thanks. Ok for (1), but how to for (2) I mean as soon I have exported the json, shall I have to copy in the postman folder each api json export? Grazie!

**Comment by ptramonti** [comment]
You have to open Postman and:
1. Create a folder inside your Api collection
2. Move the webapi that you would like to generate documentation, inside the folder (you could also organize the webapi in many folder and subfolder)
3. Export your collection in v1 format.

The webapi that are in the root level of the collection (not inside a folder) are not showed in the generated documentation.
Paolo

**Comment by loretoparisi** [comment]
@ptramonti thank you it works in this way!

**Comment by aubm** [comment]
For the record, I just released [0.8.0](https://github.com/aubm/postmanerator/releases/tag/v0.8.0).
Using this version and updating the default theme, you will not be forced to use folders anymore.
Collection V2 is still unsupported yet.

You can update the default theme by doing:

```
postmanerator themes delete default
postmanerator themes get default
```
